tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -85,9 +85,8 @@ def launch(self):
         Wait(self.marionette).until(
             expected.element_present(*self._app_loaded_locator))
 
-        # this is located at the end of the page.  If this is shown, everything is rendered.
-        Wait(self.marionette).until(
-            expected.element_present(*self._bluetooth_menu_item_locator))
+        # Wait for airplane mode menu-item to become enabled
```

Comment:
Something to do with imagecompare, from what I understood from No-Jun. I'll wait on his comment then.